addons/account/__init__.py

```diff
@@ -5,11 +5,14 @@
 
 import wizard
 import report
-from openerp import SUPERUSER_ID
+
+from odoo import api, SUPERUSER_ID
+
 
 def _auto_install_l10n(cr, registry):
     #check the country of the main company (only) and eventually load some module needed in that country
-    country_code = registry['res.users'].browse(cr, SUPERUSER_ID, SUPERUSER_ID, {}).company_id.country_id.code
+    env = api.Environment(cr, SUPERUSER_ID, {})
+    country_code = env['res.users'].sudo().browse(SUPERUSER_ID).company_id.country_id.code
```

Comment:
`env.user` should suffice here (the env is created via superuser, and `env.user` is browsed as superuser)